database/database.go

```diff
@@ -51,7 +51,7 @@ func (db *Database) RecordFingerprintImportedIntoGnuPG(newFingerprint fpr.Finger
 	}
 
 	allFingerprints := append(existingFingerprints, newFingerprint)
-	databaseMessage := makeDatabaseMessageFromFingerprints(deduplicate(allFingerprints))
+	message := makeMessageFromFingerprints(deduplicate(allFingerprints))
```

Comment:
I feel renaming all these internal instances of `databaseMessage` to just `message` have made it less clear what's happening. First impressions are that it's now tricky to grok whether we're looking at a `databaseMessage` or one of its sub messages.

Reply:
Come to think of it it feels pretty strange referring to it as a *message* at all, where it's actually a *database*